pkg/manager/member/tikv_scaler.go

```diff
@@ -94,6 +94,14 @@ func (tsd *tikvScaler) ScaleIn(meta metav1.Object, oldSet *apps.StatefulSet, new
 
 	switch meta.(type) {
 	case *v1alpha1.TidbCluster:
+		tc ,_ := meta.(*v1alpha1.TidbCluster)
+		storesInfo, err := controller.GetPDClient(tsd.pdControl, tc).GetStores()
```

Comment:
should be `tsd.deps.PDControl` ?

Reply:
Can we add `status.tikv.storeCount` and sync it in the `syncTidbClusterStatus`, so here we can just retrieve the info from the `status.tikv` because the `GetStores` will be called in `syncTidbClusterStatus` just before the scaling function.